i_scene_cp77_gltf/__init__.py

```diff
@@ -59,6 +60,10 @@ class CP77Import(bpy.types.Operator,ImportHelper):
         description="Texture Format",
         default="png")
     exclude_unused_mats: BoolProperty(name="Exclude Unused Materials",default=True,description="Enabling this options skips all the materials that aren't being used by any mesh")
+    
+    #Kwek: QoL option to match WolvenKit GUI options
+    exclude_all_mats: BoolProperty(name="Ignore All Materials",default=False,description="Import mesh only, ignoring material.json files")
```

Comment:
Note for later (no need to fix): Been awhile since I've done any updates to the open dialog, but if we have two material options, we should probably consider making a header for it.

`exclude_unused_mats` will also depend on whether or not `exclude_all_mats` is on or not.

Reply:
That's a good point. I'll keep that in mind, and maybe add another PR for that, since layout seems like an issue all in itself.

Reply:
..plus we may need to rethink UI flow and layout if we were to have one UI element overriding another. Yeah, definitely a separate PR for the full on UI header and exclusion logic. Let's talk on discord for the details so I can start working on that one :)